xcp_d/workflow/anatomical.py

```diff
@@ -953,134 +988,236 @@ def init_anatomical_wf(
-                    (lh_32k_wm_wf, ds_wmLsurf_wf, [("out_file", "in_file")]),
+                    (
+                        lh_32k_apply_warpfield,
+                        join_lh_32k_warped_surfs,
+                        [("out_file", "in1")],
+                    ),
+                    (
+                        join_lh_32k_warped_surfs,
+                        select_lh_32k_midthick_surf,
+                        [("out", "inlist")],
+                    ),
+                    (
+                        join_lh_32k_warped_surfs,
+                        select_lh_32k_pial_surf,
+                        [("out", "inlist")],
+                    ),
+                    (
+                        join_lh_32k_warped_surfs,
+                        select_lh_32k_wm_surf,
+                        [("out", "inlist")],
+                    ),
+                    (select_lh_32k_midthick_surf, ds_midLsurf_wf, [("out", "in_file")]),
+                    (select_lh_32k_pial_surf, ds_pialLsurf_wf, [("out", "in_file")]),
+                    (select_lh_32k_wm_surf, ds_wmLsurf_wf, [("out", "in_file")]),
                 ]
             )
             workflow.connect(
                 [
                     (
                         convert_xfm2world,
-                        rh_surface_apply_affine,
+                        rh_native_apply_affine,
                         [("out_file", "affine")],
                     ),
                     (
-                        rh_surface_apply_affine,
-                        rh_surface_apply_warpfield,
+                        rh_native_apply_affine,
+                        rh_native_apply_warpfield,
                         [("out_file", "in_file")],
                     ),
                     (
                         remerge_warpfield,
-                        rh_surface_apply_warpfield,
+                        rh_native_apply_warpfield,
                         [("out_file", "forward_warp")],
                     ),
                     (
                         remerge_inv_warpfield,
-                        rh_surface_apply_warpfield,
+                        rh_native_apply_warpfield,
                         [("out_file", "warpfield")],
                     ),
                     (
-                        right_sphere_raw_mris,
+                        rh_sphere_raw_mris,
                         surface_sphere_project_unproject_rh,
                         [("converted", "in_file")]
                     ),
                     (
                         surface_sphere_project_unproject_rh,
-                        rh_32k_midthick_wf,
-                        [("out_file", "current_sphere")],
-                    ), (
-                        surface_sphere_project_unproject_rh,
-                        rh_32k_pial_wf,
+                        rh_32k_resample_wf,
                         [("out_file", "current_sphere")],
+                    ), 

```

Comment:
```suggestion
                    ),
```
The linter flagged trailing whitespace.